packages/cyberstorm/README.md

```diff
@@ -0,0 +1,74 @@
+# Cyberstorm UI library
+
+## Storybook
+
+[Storybook](https://storybook.js.org/docs/react/get-started/introduction)
+provides a sandbox to build UI components in isolation, without having to start
+up the whole service stack. Additionally it showcases the existing components,
+promoting reusability.
+
+To start Cyberstorm Storybook, run `yarn workspace @thunderstore/cyberstorm-storybook storybook`.
+
+Storybook can then be accessed at [http://localhost:6006/].
+
+## Naming conventions
+
+### Basic information
+
+1. Every element targeted by CSS has to have a CSS class.
+   - Every element NOT targeted by CSS does not HAVE to have a CSS class (but can).
+   - This makes the selectors stay flat
+
+2. CSS ClassNames should follow camelCase format, e.g.:
+   - `.root`
+   - `.label`
+   - `.categoryWrapper`
+3. Do not prefix any class names with the component's name (except the color scheme variations).
+
+4. Component's root element should have the class name of `.root`
+
+5. Color scheme Variation declarations should be separated from the class by two underscores, e.g.:
+   - `.packageCard__default`
+   - `.metaItem__last`
+   - `.button__specialGreen`
+
+6. Prop names follow pascalCase:
```

Comment:
`pascalCase` and `camelCase` are used interexchangeably in this doc (2. refers to the same naming as `camelCase`, where here it's called `pascalCase`).

These should probably be kept consistent? I believe `camelCase` is generally more ambiguous if I remember correctly, and what we're after has a less ambiguous name of `lower camel case` that could be used.

Reply:
changed to camelCase. PascalCase requires the first letter of the first word to also be capitalized, which wasn't correct.